packages/backend-defaults/src/entrypoints/scheduler/lib/PluginTaskSchedulerImpl.ts

```diff
@@ -147,6 +152,47 @@ export class PluginTaskSchedulerImpl implements SchedulerService {
     return this.allScheduledTasks;
   }
 
+  getRouter(): express.Router {
+    const router = Router();
+
+    router.get('/.backstage/scheduler/v1/tasks', async (_, res) => {
+      const globalState = await TaskWorker.taskStates(
+        await this.databaseFactory(),
+      );
+
+      const tasks = new Array<TaskApiTasksResponse>();
+      for (const task of this.allScheduledTasks) {
```

Comment:
Hmm, bit torn on showing local tasks in this API at all, might be better to just stick to the global ones?

Reply:
I don't think so - wouldn't that be artificially limiting? Again given a a `coreServices.systemMetadata` you'd see how each individual host is doing and whether it's been failing some important recurring task.